### PR TITLE
add modal closebutton and unclosable modal

### PR DESCRIPTION
**Pull request by HSallinen**

### CODE REVIEW

File: client/src/languages/se.json [hunk 1]
```diff
@@ -55,6 +55,8 @@
     "mismatch" : "Lösenorden stämmer inte överens!",
     "wrong_password" : "Fel krypteringslösenord!",
 
+    "close" : "stäng",
```
Comment (dkristia) [inline client/src/languages/se.json:58]: Capital letter needed

File: client/src/languages/fi.json [hunk 2]
```diff
@@ -49,6 +49,8 @@
     "mac_instruction_2" : "Paina hiiren oikeaa näppäintä ja valitse 'Compress/Pakkaa'",
     "mac_instruction_3" : "Tiedosto ilmestyy samaan kansioon",
 
+    "close" : "sulje",
```
Comment (dkristia) [inline client/src/languages/fi.json:52]: Capital letter needed

File: client/src/languages/en.json [hunk 3]
```diff
@@ -49,6 +49,8 @@
     "mac_instruction_2" : "Right-click and click 'Compress'",
     "mac_instruction_3" : "The zipped file will appear in the same folder",
 
+    "close" : "close",
```
Comment (dkristia) [inline client/src/languages/en.json:52]: Capital letter needed

**Review by dkristia — APPROVED** [review]
thumbs up emoji

**Review by joksupoksu — APPROVED** [review]

**Pull request merged**